PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/function/runner/core/FunctionUtils.java

```diff
@@ -142,8 +143,7 @@ public static void cleanUpStagingFolder(File stagingFolder) {
                 return false;
             }
             final GlobalSearchScope scope = GlobalSearchScope.moduleWithLibrariesScope(m);
-            final PsiClass ecClass = JavaPsiFacade.getInstance(project).findClass(AZURE_FUNCTION_ANNOTATION_CLASS,
-                                                                                  scope);
+            final PsiClass ecClass = ReadAction.compute(() -> JavaPsiFacade.getInstance(project).findClass(AZURE_FUNCTION_ANNOTATION_CLASS, scope));
```

Comment:
We can use `AzureTaskManager#read`, or `#readAsObservable` which handles errors automatically.